snap/snapcraft.yaml

```
@@ -0,0 +1,53 @@
+name: ratings
+base: core22
+version: '1.4'
+license: GPL-3.0
+summary: Ubuntu App Ratings Service
+description: |
+  Backend service to support application ratings in the new Ubuntu Software Centre.
+
+grade: stable
+confinement: strict
+
+apps:
+  ratings-svc:
+    command: ratings_wrapper
+    daemon: simple
```

```suggestion
    daemon: simple
    install_mode: disable
```

This will stop snapd from auto-starting the daemon when the snap is installed, which is important because the user will need to set a valid postgresql connection str before starting it.